template/src/navigation/GuestNav.tsx

```diff
@@ -19,7 +17,6 @@ export const GuestNav = (): ReactElement => {
   return (
     <Stack.Navigator headerMode="none" initialRouteName="Intro">
       <Stack.Screen name="Intro" component={IntroScreen} />
-      <Stack.Screen name="Onboard" component={OnboardingNav} />
```

Comment:
This wasn't being used anywhere but not sure if there was another reason it was in here